appveyor.yml

```diff
@@ -0,0 +1,19 @@
+environment:
+  matrix:
+    - nodejs_version: "0.8"
```

Comment:
At least on Travis, [`npm` needs to be upgraded](https://github.com/tschaub/mock-fs/blob/6406c5fca6649c6721fa09cfa13f76895ce9d2ea/.travis.yml#L7-L9) so that things work with dependencies using `^`.  Will that be an issue on AppVeyor as well?